lib/internal/errors.js

```diff
@@ -116,6 +117,11 @@ const prepareStackTrace = (globalThis, error, trace) => {
   // Error: Message
   //     at function (file)
   //     at file
+  const code = 'code' in error ? error.code : null;
```

Comment:
If `'code' in error === false`, then we know that `error.code === undefined`. This is equivalent to:
```suggestion
  const code = error.code ?? null;
```
But, it could also simply be:
```suggestion
  const { code } = error;
```

Reply:
Because we could be dealing with user errors, I didn't want to be eagerly triggering getters unnecessarily here.

Reply:
Makes sense. Do you want to add a comment?